python/pyproject.toml

```diff
@@ -18,12 +18,12 @@ dependencies = ["requests", "tqdm", "numpy", "IPython", "setproctitle"]
 [project.optional-dependencies]
 runtime_common = ["aiohttp", "decord", "fastapi",
     "hf_transfer", "huggingface_hub", "interegular", "modelscope",
-    "orjson", "outlines>=0.0.44,<0.1.0",
+    "orjson", "outlines==0.1.11",
     "packaging", "pillow", "prometheus-client>=0.20.0",
     "psutil", "pydantic", "python-multipart",
     "pyzmq>=25.1.2", "torchao>=0.7.0", "gemlite", "uvicorn", "uvloop",
     "xgrammar>=0.1.6"]
-srt = ["sglang[runtime_common]", "torch", "vllm>=0.6.3.post1,<=0.6.4.post1", "cuda-python", "flashinfer==0.1.6"]
```

Comment:
I did not upgrade directly to v0.6.4.post1 and kept vllm 0.6.3.post1 because some major clients still need to use torch 2.4, making it inconvenient to upgrade, so I made it compatible. Moreover, we only used the quantization part of vllm, and the upgrade to vllm 0.6.5 does not provide additional benefits for sglang.

Reply:
vllm 0.6.3.post1 uses torch 2.4
vllm 0.6.4.post1 uses torch 2.5
vllm 0.6.5 uses torch 2.5

Reply:
Thanks for the clarification! Sticking with vllm 0.6.3.post1 for torch 2.4 compatibility makes sense. I'll close the PR.